x-pack/test/common/utils/server_route_repository/create_admin_service_from_repository.ts

```diff
@@ -97,6 +98,18 @@ export async function getAdminApiClient<TServerRouteRepository extends ServerRou
   };
 }
 
+export async function getCustomRoleApiClient<TServerRouteRepository extends ServerRouteRepository>(
+  st: ReturnType<typeof RoleScopedSupertestProvider>
+): Promise<RepositorySupertestClient<TServerRouteRepository>> {
+  return await getApiClient(st, 'customRole');
+}
+
+export async function getAdminApiClient<TServerRouteRepository extends ServerRouteRepository>(
+  st: ReturnType<typeof RoleScopedSupertestProvider>
+): Promise<RepositorySupertestClient<TServerRouteRepository>> {
+  return await getApiClient(st, 'admin');
+}
+
```

Comment:
I didn't want to change all the tests so I kept `getAdminApiClient` but one could argue we could have only one `getApiClient(st, "role")` instead... let me know if I should consolidate this or if it's fine like this

Reply:
I'm in favor of keeping the function split by role, it makes it much easier to audit later where we are using which role by a simple reference check